docs/fastdds/transport/udp/udp.rst

```diff
@@ -0,0 +1,100 @@
+.. _transport_udp_udp:
+
+UDP Transport
+=============
+
+UDP is a connectionless transport, where the receiving :ref:`dds_layer_domainParticipant` must open an UDP port
+listening for incoming messages, and the sending :ref:`dds_layer_domainParticipant` sends messages to this port.
+
+.. _transport_udp_enabling:
+
+Enabling UDP Transport
+----------------------
+
+Fast DDS enables an UDP transport by default.
+Nevertheless, the application can enable other UDP transports if needed.
+To enable a new UDP transport in a :ref:`dds_layer_domainParticipant`, first
+create an instance of :ref:`transport_udp_v4transportDescriptor` (for UDPv4) or
+:ref:`transport_udp_v6transportDescriptor` (for UDPv6), and add it to the user transport list of the
+:ref:`dds_layer_domainParticipant`.
+
+The examples below show this procedure in both C++ code and XML file.
```

Comment:
Now that the sections have been swapped, do you still feel that it belongs to the other section (now the previous one)?  It would be odd to show how to add the transport before talking about it